src/SConscript

```diff
@@ -276,13 +276,27 @@ imgfmtdll_files = []
 qt_imagesformats = depends.Qt.enabled_imageformats(build)
 
 if qt5:
-        suffix = 'd.dll' if build.build_is_debug else '.dll'
```

Comment:
I think this line is more succinct than the the below if statements -- you can do in 2 lines with what takes 7 below :).